sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -240,6 +241,9 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
           rewrittenResultExpressions,
           planLater(child))
 
+      case Deduplication(keys, child) =>
```

Comment:
same thought as below. this is not really aggregation so should be a different strategy.